ross/rotor_assembly.py

```diff
@@ -1208,6 +1208,18 @@ def plot_rotor(self, nodes=1, ax=None, bk_ax=None):
 
         Examples:
         """
+
+        # check slenderness ratio of beam elements
+        SR = np.array([])
+        for shaft in self.shaft_elements:
+            if shaft.sld_ratio < 30:
+                SR = np.append(SR, shaft.n)
+        if len(SR) != 0:
+            print(
+                "Warning: The beam elements " + str(SR) + " have slenderness\n"
```

Comment:
I think that here it would be better to use warning instead of a print statement:

```python
from warning import warn

...
if len(sr) != 0:
    warn(...)